events/2022-moveit-community-meeting/index.markdown

```diff
@@ -1,35 +1,45 @@
 ---
 author: Sean Kolk
 comments: false
-date: 2022-01-10 10:00:00+00:00
+date: 2022-01-25 10:00:00+00:00
 layout: page
 slug: 2022-moveit-community-meeting
 title: 2022 MoveIt Community Meeting
 ---
+<div class='row'>
+  <div class='col-sm-12 col-lg-6'>
+    <h1>2022 MoveIt Community Meeting</h1>
+    <p class="paragraph-big">
+    The 2022 MoveIt Community is a time for the maintainers, enthusiasts, and experimenters with MoveIt to meet, discuss, and decide the future of MoveIt. The event will be 5 hours with speakers, decision groups, and exhibitions to show MoveIt projects to the community. The event will be held virtually, with details to be determined.
+    </p>
+  </div>
+  <div class='col-sm-12 col-lg-6'>
+    <img style="max-width:100%" alt="Support" src="/assets/images/moveit_community_meeting_2022.png">
+  </div>
+</div>
 
-## 2022 MoveIt Community Meeting
-The 2022 MoveIt Community is a time for the maintainers, enthusiasts, and experimenters with MoveIt to meet, discuss, and decide the future of MoveIt. The event will be 5 hours with speakers, decision groups, and exhibitions to show MoveIt projects to the community. The event will be held virtually, with details to be determined.
 
 
-### Time and Venue
+## Time and Venue
 
-* Date: Thursday, Feb 24, 2022
-* Venue: (virtual)
+* **Date: Thursday, Feb 24, 2022**
+* **Venue: (virtual)**
 
-### Organizers
+## Organizers
 
 * Mark Moll (PickNik)
 * Henning Kayser (PickNik)
 * Tyler Weaver (PickNik)
 
 
-### Program (Tentative)
+## Program (Tentative)
 
 All times are UTC-6 (US Mountain  Time)
```

Comment:
Mountain time is UTC-7 until March 13th
https://en.wikipedia.org/wiki/File:World_Time_Zones_Map.png
```suggestion
All times are UTC-7 (US Mountain  Time)
```